spring-cloud-gcp-samples/spring-cloud-gcp-sql-sample/pom.xml

```diff
@@ -13,10 +13,20 @@
 	<name>Spring Cloud GCP SQL MySQL Starter Code Sample</name>
 
 	<dependencies>
+		<!-- You only need to include the starter dependency of your database engine. -->
```

Comment:
Maybe explain why both are included in the sample and whether there is any harm in that?

Reply:
So both are included actually just to get the Test to work on both postgres and mysql. From the app's perspective, It turns out that including both means it defaults to using the MySql starter when the app is run.